app/src/main/res/layout/administrator_controls_account_actions_view.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
```

Comment:
Remove this. This is not getting used.

Reply:
@rt4914 I cannot remove this `<data>` tag and `ViewModel `implementation inside it because otherwise it will give error while registering it in `MultiTypeBuilder ` in `setViewModel ` parameter as there won't be any `ViewModel ` class, associated with the corresponding `ViewBinding `class.